lantern-common

```diff
@@ -1 +1 @@
-Subproject commit 3c1f3cbee54d21e08e952eb53b90382c90c69616
+Subproject commit 81565de582437ce8d987ba1b2cc2d24ccf9d9d3c
```

Comment:
This captures https://github.com/getlantern/lantern-common/commit/81565de582437ce8d987ba1b2cc2d24ccf9d9d3c